code/script.js

```diff
@@ -1,9 +1,75 @@
-// A function that adds and remove the class "active" on the section you click on.
-// We haven't really talked about ´this´ yet, but we will... ;)
-// console.log(this) to see how it works
-function toggle() {
-  this.classList.toggle("active")
-}
-
-// Selects an HTML element, and calls a function which will be executed when the element is clicked.
-document.getElementById("section1").onclick = toggle
+const questionHeader1 = document.getElementById("question-header1")
+const questionContent1 = document.getElementById("question-content1")
+
+const questionHeader2 = document.getElementById("question-header2")
+const questionContent2 = document.getElementById("question-content2")
+
+const questionHeader3 = document.getElementById("question-header3")
+const questionContent3 = document.getElementById("question-content3")
+
+const questionHeader4 = document.getElementById("question-header4")
+const questionContent4 = document.getElementById("question-content4")
+
+const questionHeader5 = document.getElementById("question-header5")
+const questionContent5 = document.getElementById("question-content5")
+
+const questionHeader6 = document.getElementById("question-header6")
+const questionContent6 = document.getElementById("question-content6")
+
+
+
+questionHeader1.addEventListener('click', () => {
```

Comment:
For now it's acceptable to write this kind of code, I just want you to know that in the future we will learn how to construct universal algorithms so we don't have to be so repetitive :) So, do not get used to writing logic this way